ONSdigital/rm-case-service
# Introduce sampleUnitId

# Motivation and Context
Added sampleUnitId to data flowing and made partyId optional.  Added integration test to cover this.

# What has changed
- docker-compose.yml added to support ITs
- added integration test
- populated sampleUnitId and removed assumption partyId will be present

# How to test?
- run integration tests
- monitoring the rabbit queues during the tests will show that a message past in with a sampleUnitId and no partyId will result in a message out containing the same sampleUnitId

Apologies for the megadiff - I ended up merging in the reformatting work with many conflicts.  What you see below is intellij conspiring with google style to make all our lives difficult :-)
